app/src/main/java/com/everest/moviedb/network/MovieRepository.kt

```diff
@@ -0,0 +1,44 @@
+package com.everest.moviedb.network
+
+import com.everest.moviedb.models.Movie
+import com.everest.moviedb.utils.IMAGE_BASE_URL
+
+
+class MovieRepository(
+    private val retrofitClient: RetrofitClient,
```

Comment:
Instead of client, you can pass service directly, same for below also. instead of passing whole db instance we should pass Dao.